src/components/dialog.css

```diff
@@ -24,12 +24,52 @@
  * The maximum width allowed for the dialog is 480 px. Default width
  * is 100% (for container width < 480 px).
  */
-@media (min-width: 480px) {
+@media (max-width: 640px), (max-height: 640px) {
   .swg-dialog,
   .swg-toast {
     width: 480px !important;
```

Comment:
Hmm. What if the viewport is less than 480px wide? If covered by the other media rules, please add a comment here. E.g. this property will be further restricted by X rule.

Reply:
Done